plugins/inputs/gnmi/gnmi.go

```diff
@@ -304,6 +317,15 @@ func init() {
 	inputs.Add("cisco_telemetry_gnmi", New)
 }
 
+func (c *GNMI) Init() error {
+
+	// Check vendor_specific options configured by user
+	if err := choice.CheckSlice(c.VendorSpecific, supportedExtensions); err != nil {
+		return fmt.Errorf("Unsupported vendor_specific option': %w", err)
```

Comment:
Lowercase error messages
```suggestion
		return fmt.Errorf("unsupported vendor_specific option: %w", err)
```